src/styles/_common.scss

```diff
@@ -353,28 +355,39 @@ i {
         text-transform: uppercase;
         font-size: 22px;
       }
-      p {
-        margin: 20px 20px 60px;
-        font-size: 20px;
-        font-family: $medium;
+      div {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        margin-top: auto;
+        p {
+          margin: 20px;
+          font-size: 20px;
+          font-family: $medium;
+        }
+        img {
+          width: 40px;
+          height: 40px;
+          margin: 20px;
+        }
       }
       &:nth-last-child(2) {
         display: none;
         @media (max-width: 1040px) and (min-width: 701px) {
           display: block;
         }
       }
-      &:not(:last-child) {
-        a {
-          position: absolute;
-          right: 20px;
-          bottom: 20px;
-          img {
-            width: 40px;
-            height: 40px;
-          }
-        }
-      }
+      // &:not(:last-child) {
```

Comment:
I don't fully understand your use of &:not(:last-child), but I can see that this a tag links to the arrow items which are aligned 20px from the bottom right of the page. This particular arrangement (using position absolute) meant that the dates of the posts and the arrow element at the bottom of the panel weren't consistent, so I changed the rules so that they always relied on justify content:space between, and used a margin:auto to push the new div to the bottom of the panel!